python/common/org/python/types/Str.java

```diff
@@ -384,6 +384,27 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
     @org.python.Method(
             __doc__ = ""
     )
+    public org.python.Object __floordiv__(org.python.Object other) {
+        if (other instanceof org.python.types.Complex) {
```

Comment:
Hum... I see, so instead for me doing this only in Str, I should do on Object. I'll try do this and commit when I get any success!!!